app/Http/Controllers/API/AuthController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Services\UserService;
+use App\Http\Requests\UserRequest;
+use App\Http\Controllers\Controller;
+use App\Http\Resources\UserResource;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class AuthController extends Controller
+{
+    protected $userService;
+
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    public function login(UserRequest $request): JsonResponse
+    {
+        $data = $this->userService->authenticateUser($request);
+
+        return response()->json([
+            'access_token' => $data->createToken('authToken')->plainTextToken,
```

Comment:
@sabinshrestha8 You have created service classes for which I suppose you will have n-tier architecture, so just use the controller as a presentation layer. Move the token creation in service as well
https://medium.com/swlh/designing-a-multi-layered-architecture-for-building-restful-web-services-with-spring-boot-and-a12ef85b77c9